domino-ui/src/main/java/org/dominokit/domino/ui/animations/Animation.java

```diff
@@ -39,11 +39,11 @@ public static Animation create(HTMLElement element) {
         return new Animation(element);
     }
 
-    public static Animation create(BaseDominoElement element) {
+    public static <E extends HTMLElement, T extends IsElement<E>> Animation create(BaseDominoElement<E, T> element) {
         return new Animation(element.element());
     }
 
-    public static Animation create(IsElement element) {
+    public static <E extends HTMLElement> Animation create(IsElement<E> element) {
         return new Animation(element.element());
     }
 
```

Comment:
we can change this to 
`public static  Animation create(IsElement<? extends HTMLElement> element) {` 
or 
`public static  Animation create(IsElement<?> element) {` 

and remove the other create method that takes `BaseDominoElement`